ml-agents/mlagents/trainers/sac/trainer.py

```diff
@@ -84,15 +84,22 @@ def __init__(
         self.optimizer: SACOptimizer = None  # type: ignore
 
         self.step = 0
-        self.train_interval = (
-            trainer_parameters["train_interval"]
-            if "train_interval" in trainer_parameters
+
+        # Don't count buffer_init_steps in steps_per_update ratio, but also don't divide-by-0
+        self.update_steps = max(1, self.trainer_parameters["buffer_init_steps"])
+        self.reward_signal_update_steps = max(
+            1, self.trainer_parameters["buffer_init_steps"]
+        )
+
+        self.steps_per_update = (
+            trainer_parameters["steps_per_update"]
+            if "steps_per_update" in trainer_parameters
             else 1
```

Comment:
Noting your docs above about recommended ranges, would something in that range be a better default?  Even better, this could be put into a constant somewhere.